Tests/ListVolumesTests.m

```diff
@@ -61,7 +61,7 @@ - (void)testMultipleVolume_UseFirstVolume {
     XCTAssertNil(listVolumesError, @"Error listing volume paths");
     XCTAssertNotNil(volumePaths, @"No paths returned");
     XCTAssertEqual(volumePaths.count, 5, @"Wrong number of volume paths listed");
-    XCTAssertTrue([expectedVolumePaths isEqualToArray:volumePaths],
+    XCTAssertTrue([[NSSet setWithArray:expectedVolumePaths] isEqual:[NSSet setWithArray:volumePaths]],
```

Comment:
As mentioned, if you sort the results of `listVolumePaths` then you can revert these changes to the unit tests.